dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/newTimer/MyCDIInterceptor.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.ejb.apps.newtimer;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.annotation.Priority;
+import javax.annotation.Resource;
+import javax.inject.Inject;
+import javax.interceptor.AroundTimeout;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+@MyCDIInterceptorBinding
+@Interceptor
+@Priority(1)
+public class MyCDIInterceptor {
+
+    public static boolean interceptorCalled = false;
+
+    @AroundTimeout
+    public Object invoke(InvocationContext context) throws Exception {
+        interceptorCalled = true;
+        return context.proceed();       
```

Comment:
Trailing whitespace on this line.

Surprised this wasn't removed by the IDE automatically.